src/oncall/healthcheck.py

```diff
@@ -4,6 +4,9 @@
 from falcon import HTTPNotFound, HTTPInternalServerError
 from . import db
 
+import logging
+logger = logging.getLogger('oncall.app')
```

Comment:
I think this should be `logger = logging.getLogger('oncall.healthcheck')` right?

Reply:
a shorthand for the above ^  would be `logging.getLogger(__name__)` but maybe the logger named "oncall.app" is handled in a specific way?